src/common/src/session_config/mod.rs

```diff
@@ -38,6 +39,7 @@ const CONFIG_KEYS: [&str; 10] = [
     "MAX_SPLIT_RANGE_GAP",
     "SEARCH_PATH",
     "TRANSACTION ISOLATION LEVEL",
+    "QUERY_EPOCH",
```

Comment:
- With option 1, user can get an exact epoch, then still use it via QUERY_EPOCH, right?
- To implementation option 2, the frontend need to be aware of backup, thus can choose an exact hummock version to query based on given timestamp. It's not feasible to pass the query epoch from frontend and let CNs choose best-fit hummock version themselves, because CNs may have different backup caches and result in different hummock versions even given the same epoch.

Reply:
> I think both options are fine.
> 
> * With option 1, user can get an exact epoch, then still use it via QUERY_EPOCH, right? Or shall we use another ambiguous session config, e.g. QUERY_BACKUP_EPOCH.
> * To implementation option 2, frontend need to be aware of (and sync) backup manifest, thus it can choose a best-fit query epoch for the given arbitrary <timestamp>.

Sorry for the late reply. For option 1, I mean we provide a way to get the backed-up epochs via SQL statement (preferred) or risectl. Then we can set QUERY_EPOCH accordingly.

Reply:
Followed option 1. 
Added a new system table: rw_catalog.rw_meta_snapshot. Query it results all available backups along with human-readable timestamp range of them (`safe_epoch_ts` and `max_committed_epoch_ts`).
Because the size of backup manifest is small enough, I implement make frontend query it directly from meta node. 
Alternatively we can also make frontend read backup manifest from backup storage instead of meta node, but it doesn't bring significant benefit now.
```
dev=> select * from rw_catalog.rw_meta_snapshot;
 meta_snapshot_id | hummock_version_id |    safe_epoch    |      safe_epoch_ts      | max_committed_epoch | max_committed_epoch_ts  
------------------+--------------------+------------------+-------------------------+---------------------+-------------------------
                1 |                  5 |                0 |                         |    3551844974919680 | 2022-12-19 06:40:53.255
                2 |                  9 | 3551845695750144 | 2022-12-19 06:41:04.254 |    3551847139508224 | 2022-12-19 06:41:26.284
(2 rows)
```

Reply:
Currently backup creation and deletion is still via risectl. We can also support `create backup` and `delete backup <id>` SQL later.